modules/ROOT/pages/inline-css.adoc

```diff
@@ -37,75 +37,154 @@ tinymce.init({
 
-* A CSS stylesheet that is specified in the `<head>` of the page i.e. `<link rel="stylesheet" href="styles.css">`
-** Adding a stylesheet in {productname}, can be done by using the `content_css` option which automatically creates the `<link>` tag in the <head> of the page.
-*** It is recommended to not use the `content_css` option though and just **manually** include the stylesheets.
-*** The user can also manually insert a stylesheet link in the <head> of the page.
-* A CSS stylesheet that is specified in the `<body>` of the page i.e. `<link rel="stylesheet" href="styles.css">`
-** This is not recommended by https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link[MDN], but some browsers do allow it
-* A `<style>` in the `<head>` of the page i.e. `<style>p { color: red; }</style>`
-** To add a `<style>` in {productname} is by using the `content_style` option which automatically creates the `<style>` tag in the <head> of the document
-** The user can also manually put a `<style>` tag in the <head> of the page.
-* A `<style>` in the `<body>` of the page
-** As noted in the `Classic iframe` section, this should not be done, but some browsers will still allow it.
-* A `style` attribute on an {productname} element within the content i.e. inline CSS
+When a {productname} instance is in a shadow root, the {pluginame} should still support
 
-=== Shadow DOM is supported but **not** officially, with some notable limitations
+* CSS specified by the `xref:add-css-options.adoc#content_css[content_css]` option; and
 
-Shadow DOM is **not** officially supported by {productname}, but the feature will work when an editor running in `inline` or `iframe` is contained within a shadow root. 
+* CSS specified by the `xref:add-css-options.adoc#content_style[content_style]` option.
 
-When an `iframe` mode editor is contained with a shadow root, the editor content can be styled the same way, where the expected behavior for the `feature` is that it should not have to consider if the editor is in a shadow root. When an `inline` editor is contained within a shadow root, the stylesheet (`<link>` tags) and `<style>` tags are added directly as children of the shadow root and do not need to be contained within a `<head>` tag. 
 
-When in `inline mode`, the feature checks if the editor is inside a shadow root to ensure it does not try to get `<styles>` that are outside the shadow root.
+==== Classic mode
 
-When an editor is in a shadow root, the feature should still support getting the stylesheets specified via the `content_css` option as well as the CSS styles specified via  the`content_style` option.
+When a {productname} editor in classic mode is contained within a shadow root, CSS applied to the editor’s content will work as expected if the editor can also safely assume it is not operating in a shadow root.
```

Comment:
Could just be me but I don't quite understand this sentence, mainly the 'if the editor can also safely assume it is not operating in a shadow root'.